steps/install.sh

```diff
@@ -92,32 +77,22 @@ if ! pdftotext -v >/dev/null 2>&1; then
   fi
 fi
 
-if ! shellcheck --version >/dev/null 2>&1; then
+if [ ! -d /usr/local/texlive/"$(date +%Y)"/bin ]; then
   if [ -n "${linux}" ]; then
-    apt-get install -y shellcheck
+    wget https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz && \
+    zcat < install-tl-unx.tar.gz | tar xf - && \
+    rm install-tl-unx.tar.gz && \
+    cd install-tl-"$(date +%Y)"* && \
+    perl ./install-tl --scheme=e --no-interaction
+    cd .. && rm -r install-tl-"$(date +%Y)"*
   else
-    echo "Install 'shellcheck' somehow"
+    echo "Install 'texlive' somehow"
     exit 1
   fi
 fi
 
-if ! aspell --version >/dev/null 2>&1; then
-  if [ -n "${linux}" ]; then
-    apt-get install -y aspell
-  else
-    echo "Install 'GNU aspell' somehow"
-    exit 1
-  fi
-fi
-
-if ! xmlstarlet --version >/dev/null 2>&1; then
-  if [ -n "${linux}" ]; then
-    apt-get install -y xmlstarlet
-  else
-    echo "Install 'xmlstarlet' somehow"
-    exit 1
-  fi
-fi
+TEXLIVE_DIR=$(ls -d /usr/local/texlive/"$(date +%Y)"/bin/*/)
+PATH="$PATH:$TEXLIVE_DIR"
```

Comment:
@howcanunot I believe, it should be `export PATH=...` Otherwise, the changes you make will only be visible inside this script.

Reply:
done!

Reply:
@howcanunot I still think that it's wrongly placed. What if `texlive` is already installed on my machine? You still want to modify my `PATH`?

Reply:
@yegor256  well, it won't be worse
but in general it's more logical to put the `PATH` inside the if.

Reply:
fixed